std/algorithm/sorting.d

```diff
@@ -886,48 +927,8 @@ template multiSort(less...) //if (less.length > 1)
             return sort!(funs[0], ss, Range)(r);
         else
         {
-            static void multiSortImpl(funs...)(Range r) @safe
-            {
-                alias lessFun = binaryFun!(funs[0]);
-
-                static if (funs.length > 1)
-                {
-                    while (r.length > 1)
-                    {
-                        auto p = getPivot!lessFun(r);
-                        auto t = partition3!(funs[0], ss)(r, r[p]);
-                        if (t[0].length <= t[2].length)
-                        {
-                            multiSortImpl!funs(t[0]);
-                            multiSortImpl!(funs[1 .. $])(t[1]);
-                            r = t[2];
-                        }
-                        else
-                        {
-                            multiSortImpl!(funs[1 .. $])(t[1]);
-                            multiSortImpl!funs(t[2]);
-                            r = t[0];
-                        }
-                    }
-                }
-                else
-                {
-                    sort!(lessFun, ss)(r);
-                }
-            }
-
-            static bool predFun(ElementType!Range a, ElementType!Range b)
-            {
-                foreach (f; AliasSeq!(funs))
-                {
-                    alias lessFun = binaryFun!(f);
-                    if (lessFun(a, b)) return true;
-                    if (lessFun(b, a)) return false;
-                }
-                return false;
-            }
-            multiSortImpl!funs(r);
-            return assumeSorted!predFun(r);
+            multiSortImpl!(Range, ss, funs)(r);       
+            return assumeSorted!( multiSortPredFun!(Range, funs) )(r);
```

Comment:
style nitpick: no spaces between the functions - AFAICT that's the common style in this module / Phobos